src/Controller/PeopleController.php

```diff
@@ -117,4 +123,16 @@ private function getStudents(): array
         }
         return $students;
     }
+
+    private function getTeam(): array
+    {
+        $team = [];
+        $storage = json_decode($this->getStorage(), true);
+        foreach ($storage as $key => $t) {
```

Comment:
Dabar veikia, bet gal praverst požiūrio praplėtimui.
```suggestion
        foreach (array_keys($storage) as $key) {
```
Su `array_keys` funkcija iš karto gaunami visos `keys` reikšmės, tada nebereikia naudoti `$t` kintamojo, kuris jokios naudos neduoda.

Plačiau:
 * https://www.php.net/manual/en/function.array-keys.php